docs/pages/guides/create-nft.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Creating an NFT'
+---
+
+import { Steps } from 'nextra-theme-docs'
+
+# Creating an NFT
+
+## Introduction
+
+You can create new NFTs directly on the Liteflow dashboard on any collection you created using the dashboard.
+
+## Prerequisites
+
+Before creating an NFT, ensure you have:
+
+- Created the collection using the Liteflow dashboard
+- The info and image of the NFT you want to create
+- Native token on your wallet to pay for the transaction
+
+## Steps
+
+<Steps>
+
+### Access NFTs section
+
+Navigate to the [NFTs](https://dashboard.liteflow.com/nfts) section in the Liteflow dashboard.
+
+### Start creation of an NFT
+
+Click on the 'Create' button to open the form to create an NFT.
+
+### Select a Collection
+
+Choose the collection from the list.
+
+If the collection is not listed but is listed in the [Collections](https://dashboard.liteflow.com/collections) section, it means your wallet is not authorized to create an NFT on this collection. Make sure your are login with the right wallet.
```

Comment:
Maybe adding this as a warning or info box would be nice

Reply:
added